index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/style.css" type="text/css">
+    <title>Sushi Noms</title>
+</head>
+<body>
+    <header>
+        <h1>HOW MANY SUSHIS CAN YOU NOMS?</h1>
+    </header>
+        <main>
+            <section class="buttons">
+                    <h2>Feed me sushis!</h2>
+                    <button class="sushi" value="gimbap"><img src="assets/gimbap.png" alt="gimbap"></button>
+                    <button class="sushi" value="roe"><img src="assets/roe.png" alt="roe"></button>
+                    <button class="sushi" value="salshimi"><img src="assets/salshimi.png" alt="salmon sashimi"></button>
+                    <button class="sushi" value="salmon"><img src="assets/salmon.png" alt="salmon role"></button>
+                    <button class="sushi" value="tuna"><img src="assets/tuna.png" alt="tuna"></button>
+                </section>
```

Comment:
Be careful of your spacing here - the closing section tag on line 22 is indented more than the opening tag on 15 which caused a cascade of overly indented tags beneath it.